composer.json

```diff
@@ -17,16 +17,16 @@
 	],
 	"require": {
 		"php": ">=5.3.2",
-		"symfony/form": "~2.3",
-		"symfony/http-kernel": "~2.3",
-		"symfony/translation": "~2.3",
-		"symfony/yaml": "~2.3"
+		"symfony/form": "^2.6.4|~3.0",
+		"symfony/http-kernel": "^2.6.4|~3.0",
+		"symfony/translation": "^2.6.4|~3.0",
+		"symfony/yaml": "^2.6.4|~3.0"
```

Comment:
I'm still using Symfony 2.6 in a project myself, so it would be a bad idea to drop support now. :smirk: